packages/server/src/user/user.service.ts

```diff
@@ -57,59 +57,64 @@ export class UserService {
   /**
    * Find all user records from one project using project ID
    *
-   * @param projectId id of the project
+   * @param projectId ID of the project
    * @returns List of `User` object, will return empty list if no user found
    */
   async findUsersByProjectId(projectId: string): Promise<User[]> {
```

Comment:
If we have this method, may not have to keep track of a list of users in the Project schema. Having to maintain the list of users makes future actions like removing users and every "create user" action would require updating the cooresponding project